pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -240,6 +240,9 @@ const (
 	// NodeGroupNameLabel defines the label of the nodegroup name
 	NodeGroupNameLabel = "alpha.eksctl.io/nodegroup-name"
 
+	// KarpenterNameTag defines the tag of the karpenter stack name
+	KarpenterNameTag = "alpha.eksctl.io/karpenter-name"
+
```

Comment:
:thinking: do we need this? How do we tag the `fargate` and `iamserviceaccount` stacks?

Reply:
Err, no... why did I add this? :D

Reply:
WTF is this even? :DDDD

Reply:
lmao :smile:

Reply:
Pfff apparently it was used SOMEWHERE
/home/runner/work/eksctl/eksctl/pkg/cfn/manager/karpenter.go:36:7: KarpenterNameTag not declared by package v1alpha5

:D

Reply:
Oh! The get tag thingy for the cluster! That's why it's needed. Eventually, that's how it will find the stack. 🤦

Reply:
Oh I totally misread your message. 🤦

Reply:
Fargate uses special profiles then calls IsFargateEnabled with `fargateRoleExistsOnClusterStack`. 
Iamserviceaccount uses `IAMServiceAccountNameTag = "alpha.eksctl.io/iamserviceaccount-name"`.